### DISCUSSION TITLE
Is it possible to keep same iframe background for multiple slides without reloading in reveal.js?

### DISCUSSION THREAD

**Discussion by afeLeaks** [quarto-dev/quarto-cli#527] [Q&A] [answered]
I have a series of background leaflet iframes that I'd love to have work seamlessly (I have them set on timers to start from previous slide then zoom to another part of the map after <1 second).  Annoyingly the iframe refreshes each time I switch to a slide, even if the background html does not change.  Any ideas on how to get rid of the refresh when it's the same exact background html file?

**Comment by jjallaire** [comment, marked as answer]
I don't know of a way to do this. This SO answer seems to indicate that its possible but requires some pretty advanced JavaScript customization: https://stackoverflow.com/questions/61060886/prevent-reloading-of-background-iframe-in-reveal-js

**Reply by afeLeaks** [reply]
Yeah, saw that one too.  Wish the answerer would have actually wrote out how!
